syntax = "proto3";

package com.webank.ai.fate.api.mlmodel.manager;
option java_outer_classname = "ModelServiceProto";

message PublishRequest{
    string sceneId = 1;
    string myPartyId = 2;
    string partnerPartyId = 3;
    string myRole = 4;
    string commitId = 5;
    string tag = 6;
    string branch = 7;
}

message PublishResponse{
    int32 statusCode = 1;
    string message = 2;
    string error = 3;
    bytes data = 4;
}


service ModelService{
    rpc publishLoad(PublishRequest) returns (PublishResponse);
    rpc publishOnline(PublishRequest) returns (PublishResponse);
}
